sections/builder.js

```diff
@@ -2439,6 +2439,20 @@ export default [
       }
     ]
   },
+  {
+    type: "method",
+    method: "optimizerHint",
+    example: ".optimizerHint(hint)",
```

Comment:
Do you have a convention how to denote that `hint` may be either a single value or an array of values?

Reply:
Like this:
`.whereIn(column|columns, array|callback|builder)`

Reply:
@kibertoad Ok. Used `hint|hints` c12ab97